composer.lock

```diff
@@ -2438,16 +2438,16 @@
         },
         {
             "name": "drupal/core",
-            "version": "9.4.5",
+            "version": "9.5.x-dev",
```

Comment:
As I understand it, having `"prefer-stable": true` in composer.json should stop us getting dev versions when a compatible stable version (9.4.5) is available.

https://github.com/UN-OCHA/slt8-site/blob/94be95b883b0bd699e5dddd01eb7a995feb404f2/composer.json#L81

I think it's only happening in this repo.

Reply:
It wasn't only happening in this repo. After some research, I've tried with `"minimum-stability": "stable",`, which I appears to allow anything specified as less-than-stable in composer.json, which seems to be what we want. 

Will make that change on IASC and elsewhere if I see it happening.